packages/devtools/lib/src/inspector/inspector_tree.dart

```diff
@@ -156,6 +156,10 @@ abstract class InspectorTreeNode {
     return _children.length > 1 && !_children.last.isProperty;
   }
 
+  bool isDirty = true;
```

Comment:
make isDirty private exposing only a public getter if needed.

Reply:
If we are only using the setter within this class then we should make it private. That ensures that calling the dirty method is the only way isDirty is set to true. That is important as isDirty does other cleanup as well.

Reply:
We do set `isDirty` from outside of this class. In `InspectorTree._maybeClearCache()` we call `root.isDirty = false;`.

Reply:
Oh yes. That is a bit of a hack but not easily avoidable. Given that I would suggest making a setter for `isDirty` that does what
`void dirty()` does when the value is true and just sets the value to of `isDirty` to false otherwise.

Reply:
done.